pyvrp/tests/test_Population.py

```diff
@@ -166,8 +166,8 @@ def test_select_returns_same_parents_if_no_other_option():
     # small, but not zero. So let's do an experiment where we do 1000 selects,
     # and collect the number of times the parents are different.
     different_parents = 0
-    for _ in range(1_000):
-        parents = pop.select(rng)
+    for i in range(1_000):
+        parents = pop.select(rng, cost_evaluator, i == 0)
```

Comment:
Do we need the really clever `i == 0` stuff? Just updating every iteration should also be OK for tests right? If we somehow do need it, can we be more explicit with an if/else?

Reply:
Didn't do this since we removed the flag completely. For tests it is ok indeed.